src/Scan.js

```diff
@@ -276,30 +281,74 @@ class Scan extends React.Component {
       });
   }
 
+  async fetchManyRequests(items) {
+    const { mutator: { requests } } = this.props;
+    // Split the list of items into small chunks to create a short enough query string
+    // that we can avoid a "414 Request URI Too Long" response from Okapi.
+    const CHUNK_SIZE = 40;
+    const chunkedItems = chunk(items, CHUNK_SIZE);
+    const data = [];
+
+    for (const itemChunk of chunkedItems) {
+      let query = itemChunk.map(i => `itemId==${i.id}`).join(' or ');
+      query = `(${query}) and (status="Open")`;
+      requests.reset();
+      const result = await requests.GET({ params: { query, limit: 1000 } });
```

Comment:
Looks like we send requests one by one. It is better to send them in parallel for performance.

Reply:
Good catch, fixed